stdlib/asm/mem.masm

```diff
@@ -54,18 +60,64 @@ export.memcopy_words
   dropw drop drop drop
 end
 
-#! Copies an even number of words from the advice_stack to memory.
+#! Copies `n` felts from `read_ptr` to `write_ptr`.
+#!
+#! Inputs:  [n, read_ptr, write_ptr]
+#! Outputs: []
+#!
+#! Where:
+#! - n is the number of felts which should be copied.
+#! - read_ptr is the memory pointer where the felts to copy are stored.
+#! - write_ptr is the memory pointer where the felts will be copied.
+#!
+#! Total cycles: 7 + 14 * num_felts
+export.memcopy_felts
```

Comment:
I would write this using new syntax - i.e., `pub proc memcopy_felts`. This way, once https://github.com/0xMiden/miden-vm/pull/2330, this will still compile.